views/layout.jade

```diff
@@ -0,0 +1,7 @@
+doctype html
+html
+  head
+    title= title
+    link(rel='stylesheet', href='/stylesheets/style.css')
+  body
+    block content
```

Comment:
nice job, remember you can use almost the same code for your final project login and signup